unlock-app/src/hooks/useToken.ts

```diff
@@ -0,0 +1,5 @@
+import { useAppStorage } from './useAppStorage'
+
+export const useToken = () => {
+  const storage = useAppStorage()
```

Comment:
Do you think you can use the same package I've used in unlock-protocol-com site. There should be one use-localstorage-hook or something for consistency. We should get rid of the this appStorage hook at some point because it doesn't deal with some edge cases in browser refocus and storage events. 


It will also be code we don't need to maintain ourselves.